checker/tests/integration_tests.rs

```diff
@@ -114,30 +141,91 @@ fn run_directory(directory_path: PathBuf, extern_deps: Vec<(&str, String)>) -> u
             output_dir_path_buf.into_os_string().into_string().unwrap(),
         ));
     }
+    files_and_temp_dirs
+}
+
+fn build_options() -> Options {
+    // Read MIRAI options from file content.
```

Comment:
This does not seem right.

Reply:
So I actually was unclear about that comment as well; I copied it over from the code that was there (https://github.com/facebookexperimental/MIRAI/blob/676c7d0e3854c841848bfa954a0fc79dd832c867/checker/tests/integration_tests.rs#L155) but I wasn't too sure what it was referring to. 

Is there something specific you'd like me to write there or else I can come up with something?

Reply:
Just remove the comment. The code is pretty simple and clear enough.